vhdbuilder/scripts/automate_version_bump.sh

```diff
@@ -21,6 +21,29 @@ find_and_write_build_timestamp() {
     echo "$json_string" > vhdbuilder/${NEW_IMAGE_VERSION}_build_timestamp.json
 }
 
+parse_and_write_base_image_version() {
+  # Use the VHD_BUILD_ID to go through publishing-info.json
+  # If 20.04 or higher, parse the base_image_version and append it to a json file
+  # Skip for others
+  base_image_version_json_file="vhdbuilder/${NEW_IMAGE_VERSION}_base_image_version.json"
+  if [ ! -f "$base_image_version_json_file" ]; then
+    echo "{}" > "$base_image_version_json_file"
+  fi
+
+  artifact_name=$(az pipelines runs artifact list --run-id ${VHD_BUILD_ID} | jq -r '.[].name' | grep "publishing-info" | awk '/2004|2204/')
```

Comment:
nit: `artifact_names` since this is a list?

Reply:
changed